File: exercicios/javascript/IdadeEmDias.js
/*
====================================
EXERCÍCIO 2 - IDADE EM DIAS
====================================

Peça ao usuário sua idade e calcule
aproximadamente quantos dias ele viveu.
*/

const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question("Digite sua idade: ", (resposta) => {
    // int => idade informada pelo usuário (convertida de string para inteiro)
    let idade = parseInt(resposta);

    // int => cálculo dos dias vividos
    let diasVividos = idade * 365;

    console.log("Você já viveu aproximadamente " + diasVividos + " dias.");

    readline.close();
});

File: exercicios/java/IdadeEmDias.java
/*
====================================
EXERCÍCIO 2 - IDADE EM DIAS
====================================

Peça ao usuário sua idade e calcule
aproximadamente quantos dias ele viveu.
*/

import java.util.Scanner;

public class IdadeEmDias {
    public static void main(String[] args) {

        Scanner entrada = new Scanner(System.in);

        System.out.print("Digite sua idade: ");

        // int => idade informada pelo usuário
        int idade = entrada.nextInt();

        // int => cálculo dos dias vividos
        int diasVividos = idade * 365;

        System.out.println("Você já viveu aproximadamente " + diasVividos + " dias.");

        entrada.close();
    }
}
